docs/build_parameters.rst

```diff
@@ -0,0 +1,833 @@
+                },
+                "architecture": {
+                    "type": "string"
+                },
+                "enable_v1": {
+                    "type": "boolean"
+                }
+            },
+            "additionalProperties": false
+        }
+    },
+    "prefer_schema1_digest": {
+        "description": "schema 1 as preferred digest",
+        "type": "boolean"
+    },
+    "content_versions": {
+        "description": "Produce container image of given version",
+        "type": "array",
+        "items": {
+            "type": "string",
+            "enum": ["v1", "v2"]
+        }
+    },
+    "registries": {
+        "description": "Container registries to output images",
+        "type": "array",
+        "items": {
+            "type": "object",
+            "properties": {
+                "url": {
+                    "description": "Registry URI including version",
+                    "type": "string"
+                },
+                "insecure": {
+                    "description": "Don't check SSL certificate for url",
+                    "type": "boolean"
+                },
+                "auth": {
+                    "description": "Authentication information",
+                    "type": "object",
+                    "properties": {
+                        "cfg_path": {
+                            "description": "Path to directory  containing .dockercfg for registry auth",
+                            "type": "string"
+                        }
+                    },
+                    "additionalProperties": false,
+                    "required": ["cfg_path"]
+                }
+            },
+            "additionalProperties": false,
+            "required": ["url"]
+        }
+    },
+    "yum_proxy": {
+        "description": "Proxy to access yum repositories",
+        "type": "string"
+    },
+    "source_registry": {
+        "description": "Container registry to pull parent images",
+        "type": "object",
+        "properties": {
+            "url": {
+                "description": "Registry url",
+                "type": "string"
+            },
+            "insecure": {
+                "description": "Don't check SSL certificate for url",
+                "type": "boolean"
+            }
+        },
+        "additionalProperties": false,
+        "required": ["url"]
+    },
+    "sources_command": {
+        "description": "Command to retrieve artifacts in lookaside cache",
+        "type": "string"
+    },
+    "required_secrets": {
+        "description": "List of OpenShift secrets required by this configuration",
+        "type": "array",
+        "items": {
+            "type": "string"
+        }
+    },
+    "worker_token_secrets": {
+        "description": "List of OpenShift secrets for worker clusters authentication",
+        "type": "array",
+        "items": {
+            "type": "string"
+        }
+    },
+    "build_json_store": {
```

Comment:
osbs-client calls it `build_json_store`, but it actually reads from `build_json_dir` from configuration:

```
def get_build_json_store(self):                                                              
    return self._get_value("build_json_dir", GENERAL_CONFIGURATION_SECTION, "build_json_dir")
```

I'll change it to `build_json_dir` because that's what current admins are used to.

Reply:
Done.